policies.tex

```diff
@@ -45,6 +47,7 @@ \subsection{}For each position,
 \duty The club may ask individual candidates questions in private.
 \duty To be elected, a candidate must receive approval from at least a one-half vote.  
 \subsection{}Candidates must accept their nomination to run. Candidates should be present for the election meeting if possible but are not required to be. Candidates are encouraged to make themselves available to answer questions if they cannot attend in person.
+\subsection{}Candidates present for the election may vote by written ballot once the club has completed asking all questions, and their votes shall be tallied with the other club members present.
```

Comment:
replace "with the other club members present" with "with the rest of the votes."